pyproject.toml

```diff
@@ -170,6 +170,7 @@ matrix.airflow.dependencies = [
     { value = "pydantic>=1.10.0,<2.0.0", if = ["2.6"]},
     { value = "apache-airflow==2.7", if = ["2.7"] },
     { value = "apache-airflow==2.8", if = ["2.8"] },
+    { value = "pendulum<3.0.0", if = ["2.3", "2.4", "2.5", "2.6", "2.7"] },
```

Comment:
Thanks for working on this, @jbandoro !

I believe this solves the issue in the tests:
https://github.com/jbandoro/astronomer-cosmos/pull/1